apps/remote_control/lib/mix/tasks/namespace/transform/beams.ex

```diff
@@ -47,7 +47,7 @@ defmodule Mix.Tasks.Namespace.Transform.Beams do
   defp changed?(_, _), do: true
 
   defp block_until_done(same, same) do
-    Mix.Shell.IO.info("done")
+    Mix.Shell.IO.info("\ndone")
```

Comment:
```suggestion
    Mix.Shell.IO.info("\n done")
```
That way indentation matches the rest of the task's messages